src/main/java/net/minecraftforge/common/ForgeMod.java

```diff
@@ -216,6 +218,12 @@ public void missingSoundMapping(RegistryEvent.MissingMappings<SoundEvent> event)
         }
     }
 
+    @SubscribeEvent
+    public void registerLevelTypes(RegistryEvent.Register<LevelType> event)
+    {
+        LevelTypes.forEach(event.getRegistry()::register);
+    }
```

Comment:
Not sure if this is the appropriate way to do this - with there being no vanilla registry to wrap, the `minecraft` namespaced instances need to be added somewhere.

Guidance would be appreciated.